x/ecocredit/basket/client/tx.go

```diff
@@ -97,8 +107,13 @@ Flags:
 				return err
 			}
 
-			if minStartDateString != "" && startDateWindow != 0 {
-				return fmt.Errorf("both %s and %s cannot be set", FlagStartDateWindow, FlagMinimumStartDate)
+			if (minStartDateString != "" && startDateWindow != "") ||
+				(startDateWindow != "" && yearsInThePast != 0) ||
+				(minStartDateString != "" && yearsInThePast != 0) {
```

Comment:
We should also return an error if all 3 are set
```suggestion
			if (minStartDateString != "" && startDateWindow != "") ||
				(startDateWindow != "" && yearsInThePast != 0) ||
				(minStartDateString != "" && yearsInThePast != 0) ||
				(minStartDateString != "" && startDateWindow != "" && yearsInThePast != 0) {
```

Reply:
That's a duplicate check. We don't need this.

Reply:
The added line is already covered with the other checks. We don't need this.

Reply:
🤦